mmdet3d/apis/test.py

```diff
@@ -29,8 +35,38 @@ def single_gpu_test(model, data_loader, show=False, out_dir=None):
             result = model(return_loss=False, rescale=True, **data)
 
         if show:
-            model.module.show_results(data, result, out_dir)
+            if hasattr(model.module, 'show_results'):
+                model.module.show_results(data, result, out_dir)
+            else:
+                batch_size = len(result)
```

Comment:
Add comment here indicating where is this logic adopted from and what is it for.